data/json/recipes/armor/torso.json

```
@@ -239,12 +278,13 @@
     "subcategory": "CSC_ARMOR_TORSO",
     "skill_used": "fabrication",
     "difficulty": 3,
-    "time": "1 h",
+    "time": "3 h",
     "autolearn": true,
     "book_learn": [ [ "recipe_melee", 2 ] ],
     "using": [ [ "cordage", 1 ] ],
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
-    "components": [ [ [ "scrap", 80 ] ] ]
+    "components": [ [ [ "scrap", 80 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_armorsmithing", "required": false, "time_multiplier": 2 } ]
```

```suggestion
    "proficiencies": [ { "proficiency": "prof_armorsmithing", "required": false, "time_multiplier": 1.5 } ]
```
For things like this where armorsmithing might help but not blacksmithing, I'm keeping the multiplier low, just because it's impossible to learn armorsmithing from this unless you already have the requirements